requirements.txt

```diff
@@ -0,0 +1,8 @@
+cycler==0.10.0
+env==0.1.0
+kiwisolver==1.2.0
```

Comment:
Where is this used?

Reply:
Thse are basics packages installed with every virtual environnement

Reply:
We can remove them indeed. But they will reinstall automatically when using the venv std python library. I can dig deeper. I thought it was not worth it, but I will.

Reply:
- These are all matplotlib dependencies